src/main/java/com/conveyal/gtfs/GTFS.java

```diff
@@ -0,0 +1,202 @@
+        if (cmd.hasOption("load")) {
+            String filePath = cmd.getOptionValue("load");
+            loadedFeed = load(filePath, dataSource);
+            LOG.info("The unique identifier for this feed is: {}", loadedFeed);
+        }
+
+        if (cmd.hasOption("validate")) {
+            String feedToValidate = cmd.getOptionValue("validate");
+            if (feedToValidate != null && loadedFeed != null) {
+                LOG.warn("Validating the specified feed {} instead of {} (just loaded)", feedToValidate, loadedFeed);
+            }
+            if (feedToValidate == null && loadedFeed != null) {
+                feedToValidate = loadedFeed;
+            }
+            if (feedToValidate != null) {
+                LOG.info("Validating feed with unique identifier {}", feedToValidate);
+                validate (feedToValidate, dataSource);
+            } else {
+                LOG.error("No feed to validate. Specify one, or load a feed in the same command.");
+            }
+        }
+
+        if (cmd.hasOption("graphql")) {
+            Integer port = Integer.parseInt(cmd.getOptionValue("graphql"));
+            LOG.info("Starting GraphQL server on port {}", port);
+            throw new UnsupportedOperationException();
+        }
+
+    }
+
+    /**
+     * The parameter to Option.builder is the short option. Use the no-arg builder constructor with .longOpt() to
+     * specify an option that has no short form.
+     */
+    private static Options getOptions () {
+        Options options = new Options();
+        options.addOption(Option.builder("h").longOpt("help").desc("print this message").build());
+        options.addOption(Option.builder().longOpt("load").hasArg()
+                .argName("file").desc("load GTFS data from the given file").build());
+        options.addOption(Option.builder().longOpt("validate").hasArg().optionalArg(true).argName("feed")
+                .desc("validate the specified feed. defaults to the feed loaded with the --load option").build());
+        options.addOption(Option.builder("d").longOpt("database")
+                .hasArg().argName("url").desc("JDBC URL for the database. Defaults to " + DEFAULT_DATABASE_URL).build());
+        options.addOption(Option.builder("u").longOpt("user")
+                .hasArg().argName("username").desc("database username").build());
+        options.addOption(Option.builder("p").longOpt("password")
+                .hasArg().argName("password").desc("database password").build());
+        options.addOption(Option.builder().longOpt("graphql")
+                .desc("start a GraphQL API on the given port").optionalArg(true).build());
+        options.addOption(Option.builder().longOpt("json").desc("optionally store in result.json").build());
+        return options;
+    }
+
+    private static void printHelp(Options options) {
+        final String HELP = String.join("\n",
+                "java -cp gtfs-lib-shaded.jar com.conveyal.gtfs.GTFS [options]",
```

Comment:
Please add more descriptions of what the options are.

Reply:
The command line parser library will include the descriptions of the options. Here's the current output:
```
usage: java -cp gtfs-lib-shaded.jar com.conveyal.gtfs.GTFS [options]

 -d,--database <url>        JDBC URL for the database. Defaults to
                            jdbc:postgresql://localhost/gtfs
    --graphql               start a GraphQL API on the given port
 -h,--help                  print this message
    --json                  optionally store in result.json
    --load <file>           load GTFS data from the given file
 -p,--password <password>   database password
 -u,--user <username>       database username
    --validate <feed>       validate the specified feed. defaults to the
                            feed loaded with the --load option
```